measure-builds/src/main/java/com/automattic/android/measure/reporters/InternalWooCommerceA8cCiReporter.kt

```diff
@@ -22,38 +20,38 @@ import io.ktor.http.HttpHeaders
 import io.ktor.http.HttpHeaders.Authorization
 import io.ktor.http.HttpStatusCode
 import io.ktor.http.contentType
-import kotlinx.serialization.encodeToString
-import kotlinx.serialization.json.Json
-import org.gradle.api.logging.Logger
-import org.gradle.api.provider.Provider
+import kotlinx.coroutines.runBlocking
 import java.io.File
 import java.util.Locale
 import java.util.concurrent.TimeUnit.MILLISECONDS
 import java.util.concurrent.TimeUnit.MINUTES
-import kotlin.io.path.Path
-import kotlin.io.path.absolutePathString
-import kotlin.io.path.createDirectories
-import kotlin.io.path.createFile
-import kotlin.io.path.exists
-import kotlin.io.path.writeText
 import kotlin.time.Duration.Companion.seconds
 
-class MetricsReporter(
-    private val logger: Logger,
-    private val authToken: Provider<String>,
-    private val buildDir: File,
-) {
-    suspend fun report(
-        report: InMemoryReport,
-        gradleScanId: String?
+
+object InternalA8cCiReporter {
```

Comment:
In the future, we can think of extracting this to a separate plugin, to not ship a core plugin with A8c-specific reporter.